src/app/ReadHandler.h

```diff
@@ -190,13 +225,22 @@ class ReadHandler : public Messaging::ExchangeDelegate
         aMaxInterval = mMaxInterval;
     }
 
+    CHIP_ERROR SetMinReportingInterval(uint16_t aMinInterval)
```

Comment:
If this is really only there for tests, we should maybe call it SetMinReportingIntervalForTests?